clients/src/main/java/org/apache/kafka/common/requests/LeaderAndIsrRequest.java

```diff
@@ -51,7 +51,16 @@ public static class Builder extends AbstractControlRequest.Builder<LeaderAndIsrR
         public Builder(short version, int controllerId, int controllerEpoch, long brokerEpoch,
                        List<LeaderAndIsrPartitionState> partitionStates, Map<String, Uuid> topicIds,
                        Collection<Node> liveLeaders) {
-            super(ApiKeys.LEADER_AND_ISR, version, controllerId, controllerEpoch, brokerEpoch);
+            super(ApiKeys.LEADER_AND_ISR, version, controllerId, controllerEpoch, brokerEpoch, false);
```

Comment:
let's define this in terms of the other constructor to avoid duplicating the below code. something like:
```
this(version, controllerId, controllerEpoch, brokerEpoch, false)
```